languages/elixir/concepts/atoms/links.json

```diff
@@ -1,26 +1,10 @@
 [
-  {
-    "url": "https://elixir-lang.org/getting-started/basic-types.html#atoms",
-    "description": "atom"
-  },
-  {
-    "url": "https://elixir-lang.org/getting-started/basic-types.html#atoms",
-    "description": "atom"
-  },
-  {
-    "url": "https://elixir-lang.org/getting-started/basic-types.html#atoms",
-    "description": "atom"
-  },
   {
     "url": "https://elixir-lang.org/getting-started/basic-types.html#atoms",
     "description": "atom"
```

Comment:
@jwarwick Make sure to give a nice human-readable descriptions to the links, e.g. "Getting Started guide - Atoms". The description will be shown directly to the student, take a look at the screenshot in this PR: https://github.com/exercism/v3/issues/2499